WordPress/Classes/Services/PostService.m

```diff
@@ -676,6 +676,26 @@ - (void)restoreRemotePostWithPost:(AbstractPost*)post
     [remote restorePost:remotePost success:successBlock failure:failureBlock];
 }
 
+- (void)getLikesForPost:(AbstractPost *)post
```

Comment:
I don't know how you're going to call this method, but I wonder if this should be postID instead of AbstractPost. NotificationsVC and NotificationsDetailVC get the PostID from the notification. So it looks like you'd have to create an AbstractPost just to fetch likes. But that's not really necessary since fetching only requires the ID.

Reply:
Ah yes, I was actually in two minds on fetching the likes through `AbstractPost` or simply `postID` and `siteID`. My initial thought was, using `AbstractPost` would reuse most of the existing logic like `PostServiceRemoteFactory` that accepts a `Blog` object, and also makes use of the existing mocking functionalities on the unit testing side. 

As for how the caller utilizes this method, I assume the app would synchronize blogs when the app is opened; and information for the `Blog` and `AbstractPost` can be obtained through `BlogService`:
```objc
- (nullable Blog *)blogByBlogId:(NSNumber *)blogID;
``` 
and `PostService` respectively: 
```objc
- (nullable AbstractPost *)findPostWithID:(NSNumber *)postID inBlog:(Blog *)blog;
```

However, after giving it a second thought, going with just `postID` and `siteID` will make the implementation more reusable should we decide to implement the Likes page from the Reader page – since Reader utilizes a different model (i.e. `ReaderPost`). 

Thanks for your input @ScoutHarris, and I'll make the necessary adjustments shortly! 🙂